app/Http/Controllers/Auth/LoginController.php

```diff
@@ -43,7 +43,7 @@ public function authenticated()
             return redirect()->route('admin.index');
         }
         
-        return redirect()->route('blog.index');
+        return redirect()->route('user.index');
```

Comment:
why have this change ?

Reply:
user.index is dashboard. User can see all blog when login.  User will see user's blog when use blog.index.

Reply:
i mean why Comments task have change in routing of Blog Task

Reply:
If use route(blog.index), when login it'll redirect to 127.0.0.1/blog. it's a blog-manage page. So i have change route(user.index), it's dashboard for user.